source/working-groups/documentation.md

```diff
@@ -1,38 +1,87 @@
-- [2025-03-21](https://time.is/compare/1500_21_Mar_2025_in_UTC) - [Notes](https://docs.google.com/document/d/1AuGT-mGq9ssOCrHRRgQdcSVz2XbAD-qECXwBHCgMo7E/edit?usp=sharing)
-- [2025-01-17](https://time.is/compare/1400_17_Jan_2025_in_UTC) - [Notes](https://docs.google.com/document/d/1o_gjFaFpl62V8SJSbeL8V_h2-JAHrrrS48jyImU-A7c/edit?usp=sharing)
-- [2024-11-22](https://time.is/compare/1500_22_Nov_2024_in_UTC) - [Notes](https://docs.google.com/document/d/1oP3v0iHBDIsvrkPWOr524sdweCv9nfOsb6KWA-oojXo/edit?usp=sharing)
-- [2024-10-18](https://time.is/compare/1300_18_Oct_2024_in_UTC) - [Notes](https://docs.google.com/document/d/1O1v9aMSCNe9ak2_Md2plfaJw42XJuIrb7l1-sSg4hus/edit?usp=sharing)
-- [2024-09-20](https://time.is/compare/1400_20_Sep_2024_in_UTC) - [Notes](https://docs.google.com/document/d/1wUpmkOfQICuPQPbITqJ2oZSD2cSJBdHioEsaLck6_Es/edit?usp=sharing) [recording](https://harvard.zoom.us/rec/share/UBmb9Mzd_grPPsbX7jik54BGRMOHoskcVUab434u6FJOiyPE-fPN-AQjcWQmhSQ2.KAUPfDyco5S2aPzM)
+---
+
+## 🗂️ Previous Meetings & Notes
+
+| Date        | Notes           | Recording      |
+|-------------|------------------|----------------|
+| 2025-05-16  | [Notes](https://docs.google.com/document/d/1YQmixOt1E-hGWqDlTbjIkIu9zNKbm0db/edit?usp=drive_link&ouid=112670259962851928741&rtpof=true&sd=true)       |                |
+| 2025-04-18  | [Notes](https://docs.google.com/document/d/1Orv0sPTpGyuPVYnvDVuvWQe39FnxZg4S/edit?usp=drive_link&ouid=112670259962851928741&rtpof=true&sd=true)       |                |
+| 2025-03-21  | [Notes](https://docs.google.com/document/d/1AuGT-mGq9ssOCrHRRgQdcSVz2XbAD-qECXwBHCgMo7E/edit?usp=drive_link)       |                |
+| 2025-01-17  | [Notes](https://docs.google.com/document/d/1o_gjFaFpl62V8SJSbeL8V_h2-JAHrrrS48jyImU-A7c/edit?usp=drive_link)       |                |
+| 2024-11-22  | [Notes](https://docs.google.com/document/d/1oP3v0iHBDIsvrkPWOr524sdweCv9nfOsb6KWA-oojXo/edit?usp=drive_link)       |                |
+| 2024-10-18  | [Notes](https://docs.google.com/document/d/1O1v9aMSCNe9ak2_Md2plfaJw42XJuIrb7l1-sSg4hus/edit?usp=drive_link)       |                |
+| 2024-09-20  | [Notes](https://docs.google.com/document/d/1wUpmkOfQICuPQPbITqJ2oZSD2cSJBdHioEsaLck6_Es/edit?usp=drive_link)       | [Recording](#) |
+
+> Want to contribute a summary or your own notes? [Email us](mailto:docs@dataverse.org) or add a comment in the shared Google Doc!
+
+---
+
+## 📚 Current Projects
+
+- ✅ **Quickstart Guide Overhaul**  
+  Simplify the new user experience and help contributors get started faster.
+  
+- ✅ **SPA Documentation Integration**  
+  Ensure docs keep up with the newly designed Dataverse App.
+
+---
+
+## 🔗 Useful Links
+
+- 📘 [Preview Docs Site](https://preview.guides.gdcc.io/)
+- 💬 [Zulip Community Chat](https://dataverse.zulipchat.com)
+- 🧪 [Developer Docs](https://guides.dataverse.org/en/latest/)
+- 🛠️ [Open Doc Issues - Under Construction](https://guides.dataverse.org/en/latest/)
```

Comment:
For now maybe we could link to https://github.com/IQSS/dataverse/labels/Feature%3A%20User%20Guide